tests/test_training/test_opt_lab_jax.py

```diff
@@ -263,3 +264,18 @@ def cost_fn(sq):
         opt.minimize(cost_fn, by_optimizing=[sq], max_steps=100)
 
         assert math.all(og_r != sq.parameters.r.value)
+
+    def test_cat_state_optimization(self):
+        # Note: we need to intitialize the cat state with a non-zero value. This is because
+        # the gradients are zero when x is zero.
+        cat_state = Coherent(0, x=0.1, x_trainable=True) + Coherent(0, x=-0.1, x_trainable=True)
+        expected_cat = Coherent(0, x=np.sqrt(np.pi)) + Coherent(0, x=-np.sqrt(np.pi))
+
+        def cost_fn(cat_state):
+            cat_state = cat_state.normalize()
+            return -math.abs(cat_state.fidelity(expected_cat.normalize()))
+
+        opt = OptimizerJax(learning_rate=0.001, stable_threshold=1e-14)
```

Comment:
How were the `learning_rate` and `stable_threshold` determined? Are they "standard" for MrMustard? If not, is it worth adding a comment about them?